### PR TITLE
Luan

### PR DESCRIPTION
**Pull request by luanrcard**
RawMaterial class and repository name changed to InvoiceDetails (issue #36)

### CODE REVIEW

File: PCU/src/main/java/com/coffeelovers/pcu/PcuApplication.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ ApplicationRunner init(
 			ActorRepository actorRepository, 
 			VariableCostsRepository variableCostsRepository, 
 			InvoiceRepository invoiceRepository,
-			RawMaterialRepository rawMaterialRepository
+			InvoiceDetailsRepository rawMaterialRepository
```
Comment (tatacsd) [inline PCU/src/main/java/com/coffeelovers/pcu/PcuApplication.java:35]: Are we not changing the rawmaterialRepository variable name to match the class?
```suggestion
			InvoiceDetailsRepository invoiceDetailsRepository
```

File: PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java [hunk 2]
```diff
@@ -17,21 +17,21 @@
...
+import com.coffeelovers.pcu.model.InvoiceDetailsRepository;
 
 @CrossOrigin(origins = "http://localhost:8081")
 @RestController
 @RequestMapping("/api")
 public class RawMaterialController {
 	
 	@Autowired
-	RawMaterialRepository rawMaterialRepository;
+	InvoiceDetailsRepository rawMaterialRepository;
```
Comment (tatacsd) [inline PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java:29]: same here

File: PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java [hunk 3]
```diff
@@ -53,19 +52,19 @@ public RawMaterial(long invoiceID, long supplierID, String noteString) {
...
 	 */
-	public RawMaterial(long invoiceID, long supplierID, String noteString, String dateTime) {
+	public InvoiceDetails(long invoiceID, long supplierID, String noteString, String dateTime) {
 		this.invoiceID = invoiceID;
 		this.supplierID = supplierID;
 		this.noteString = noteString;
 		this.dateTime = LocalDate.parse(dateTime);
 	}
 
 	public long getRawMaterialID() {
```
Comment (tatacsd) [inline PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java:62]: and setters and getters

File: PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java [hunk 4]
```diff
@@ -53,19 +52,19 @@ public RawMaterial(long invoiceID, long supplierID, String noteString) {
...
-		return rawMaterialID;
+		return invoiceDetailsID;
 	}
 
 	public void setRawMaterialID(long materialID) {
```
Comment (tatacsd) [inline PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java:66]: setter

File: PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java [hunk 5]
```diff
@@ -105,7 +105,7 @@
 	}
 	
 	@PostMapping("/rawMaterials")
-	public ResponseEntity<RawMaterial> createRawMaterial(@RequestBody RawMaterial rawMaterial){
+	public ResponseEntity<InvoiceDetails> createRawMaterial(@RequestBody InvoiceDetails rawMaterial){
```
Comment (tatacsd) [inline PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java:108]: the variable name here too

File: PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java (hunk 2 above)
Comment (tatacsd) [inline PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java:29]:
```suggestion
	InvoiceDetailsRepository invoiceDetailsRepository;
```

File: PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java [hunk 6]
```diff
@@ -17,21 +17,21 @@
...
 @CrossOrigin(origins = "http://localhost:8081")
 @RestController
 @RequestMapping("/api")
 public class RawMaterialController {
 	
 	@Autowired
-	RawMaterialRepository rawMaterialRepository;
+	InvoiceDetailsRepository rawMaterialRepository;
 	
 	@GetMapping("/rawMaterials")
```
Comment (tatacsd) [inline PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java:31]: Should we also change the api?
```suggestion
	@GetMapping("/invoiceDetails")
```

**Review by tatacsd — CHANGES_REQUESTED** [review]

File: PCU/src/main/java/com/coffeelovers/pcu/PcuApplication.java (hunk 1 above)
Comment (Andrew-shanan) [reply PCU/src/main/java/com/coffeelovers/pcu/PcuApplication.java:35]: wait my friend, i have done all these stuff(tables, Repositories, classes.. ). no need to update any part of  backend  from your side!!

Comment (Andrew-shanan) [reply PCU/src/main/java/com/coffeelovers/pcu/PcuApplication.java:35]: No all classes and repositories are working constantly with database, so no need to work on them. Please, keep just working on front-end.
I will push what I have done after the today's class. Now I am testing all the back-end components

File: PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java (hunk 6 above)
Comment (Andrew-shanan) [reply PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java:31]: No need. it is already done on my side. ignore it please

File: PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java (hunk 2 above)
Comment (Andrew-shanan) [reply PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java:29]: No need. it is already done on my side. ignore it please

File: PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java (hunk 5 above)
Comment (Andrew-shanan) [reply PCU/src/main/java/com/coffeelovers/pcu/controller/RawMaterialController.java:108]: No need. it is already done on my side. ignore it please

File: PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java (hunk 4 above)
Comment (Andrew-shanan) [reply PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java:66]: No need. it is already done on my side. ignore it please

File: PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java (hunk 3 above)
Comment (Andrew-shanan) [reply PCU/src/main/java/com/coffeelovers/pcu/model/InvoiceDetails.java:62]: No need. it is already done on my side. ignore it please

**Comment by tatacsd** [comment]
Changes made in #54

**Pull request closed without merging**